src/operator/tensor/broadcast_reduce_op_value.cc

```diff
@@ -284,10 +284,16 @@ array.
 
 Examples::
 
-  x = [[1, 2],
-       [3, 4]]
+  x = [[[1,2],
+        [3,4]],
+       [[2,2],
+        [5,6]]]
```

Comment:
Please insert spaces just like the format before.

Reply:
i will add the spaces, but I borrowed the format here from ``sum`` operator in line ~92 and they dont have spaces.

Reply:
Then would you please fix those too?

Reply:
sure i can do that